### DISCUSSION TITLE
Crashing

### DISCUSSION THREAD

**Discussion by juanclem** [thatskymod/Sky-CotL-Scripts#71] [General]
I've been crashing everytime i use _dukun_ then unlock **emotes and cosmetics**. Including the other buttons too. It also crashes multiple times everytime i  open it ever since that **recent update** 😞.

**Comment by juanclem** [comment]
https://drive.google.com/file/d/1SPYEMLmLVnlQ782ump5yq8rz9F6PTfdc/view?usp=drivesdk

**Comment by thatskymod** [comment]
Nice catch! This is definitely something that needs to be looked into.

**Reply by Sauceboykkkk** [reply]
Hello, It has been solved?

**Comment by thatskymod** [comment]
A new release of libLangit attempts to address the issue with Cosmetics and Emotes:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libLangit022rc.so

I have successfully tested it.
Are you able to confirm this on your end?

**Reply by juanclem** [reply]
Thank you! It works properly now

**Comment by juanclem** [comment]
Thank you sm
